charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/apiserver.yaml

```diff
@@ -1739,7 +1739,7 @@ data:
                 "30d"
             ]
         },
-        "timezone": "UTC",
+        "timezone": "browser",
```

Comment:
Please see info on top of file: https://github.com/prometheus-community/helm-charts/blob/55bfd54493db3e88ad1bf6b59744149faf59fa69/charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/apiserver.yaml#L3

Reply:
Hi @monotek, thx for the hint. I found the real reason https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/447, i will close this PR and will come back to you later. 😅